Makefile

```diff
@@ -195,6 +197,8 @@ build-examples:
 	esc -pkg frontend -o examples/hotrod/services/frontend/gen_assets.go  -prefix examples/hotrod/services/frontend/web_assets examples/hotrod/services/frontend/web_assets
 ifeq ($(GOARCH), s390x)
 	$(GOBUILD) -o ./examples/hotrod/hotrod-$(GOOS)-$(GOARCH) ./examples/hotrod/main.go
+else ifeq ($(GOARCH), ppc64le)
```

Comment:
The build command for s390x and ppc64le are looking the same. Could we merge them?

Reply:
Changes made as requested.